vue/components/organisms/ripe-image/ripe-image.vue

```diff
@@ -257,7 +654,53 @@ export const RipeImage = {
             crop: this.crop,
             showInitials: this.showInitials,
             initialsGroup: this.initialsGroup,
-            initialsBuilder: this.initialsBuilder
+            initialsBuilder: this.initialsBuilder,
+            rotation: this.rotation,
+            flip: this.flip,
+            mirror: this.mirror,
+            boundingBox: this.boundingBox,
+            algorithm: this.algorithm,
+            background: this.background,
+            engine: this.engine,
+            initialsProfile: this.initialsProfile,
+            initialsProfiles: this.initialsProfiles,

```

Comment:
Adapt according to https://github.com/ripe-tech/ripe-sdk/pull/213/files#r522188738